### PR TITLE
Fixed create record bug

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js [hunk 1]
```diff
@@ -15,18 +15,20 @@ const DataCol = require('../schemaRdb/dataCol')
 
 const NodeRepository = require('../../../record/persistence/nodeRepository')
 
-const runSelect = async (surveyId, tableName, cols, offset, limit, filterExpr, sort = '', client) => {
+const runSelect = async (surveyId, tableName, cols, offset, limit, filterExpr, filterStr, sort = '', client) => {
```
Comment (minotogna) [inline server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js]: I don't like having two different parameters that do the same thing. Please use only one

Comment (sgiaccio) [reply server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js]: OK, done

File: server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ const NodeRepository = require('../../../record/persistence/nodeRepository')
 
 const runSelect = async (surveyId, tableName, cols, offset, limit, filterExpr, sort = '', client) => {
   const schemaName = SchemaRdb.getName(surveyId)
+  // Either filterExpr or filterStr need to be passed, the other one must be null
   const { str: filterQuery, params: filterParams } = filterExpr ? Expression.getWherePerparedStatement(filterExpr) : {}
   const colParams = Expression.toParamsObj(cols, 'col')
```
[the remark this answers is not in the document]
Comment (minotogna) [reply server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js]: I agree, it's better to rename this method

File: common/exprParser/expression.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const { toString: toStringUtils, isValid } = require('./helpers/utils')
 const { types } = require('./helpers/types')
 
 const operators = require('./helpers/operators')
-const { toParamsObj, getWherePerparedStatement } = require('./helpers/sqlUtils')
+const { toSqlPreparedStamentParams, getWherePerparedStatement } = require('./helpers/sqlUtils')
```
Comment (minotogna) [inline common/exprParser/expression.js]: there's a type toSqlPrepared**Stament**Params --> toSqlPrepared**Statement**Params

Comment (minotogna) [reply common/exprParser/expression.js]: ok let's postpone it

**Pull request merged**